pom.xml

```diff
@@ -74,7 +71,7 @@
       <name>Henri Yandell</name>
       <id>bayard</id>
       <email>bayard@apache.org</email>
-      <organization />
+      <organization/>
```

Comment:
@Isira-Seneviratne 
Please do not change this formatting.

Reply:
That was formatted by IntelliJ.

Reply:
It does not matter who formatted what, it's still your PR ;-)